packages/app-collective/src/Overview/types.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright 2017-2019 @polkadot/app-democracy authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { BlockNumber } from '@polkadot/types';
+
+import BN from 'bn.js';
+
+export type ElectionsInfo = {
+  members: Array<[string, BlockNumber]>,
```

Comment:
Prefer this form (which is what eslint recommended) likes - `T[]` as opposed to `Array<T>` (would be good to start using it)

```
members: [string, BlockNumber][]
candidates: string[]
```

(nit) alphabetical